src/views/Components.vue

```diff
@@ -56,6 +56,7 @@
       <tm-li-stake-transaction :transaction="Object.assign(JSON.parse(JSON.stringify(txs[4])), { state: 'locked' })" :validators="validators" unbonding_time="2592000000" v-on:end-unbonding="log('end unbonding')"/>
       <tm-li-stake-transaction :transaction="txs[4]" :validators="validators" unbonding_time="2592000000" v-on:end-unbonding="log('end unbonding')"/>
       <tm-li-stake-transaction :transaction="Object.assign(JSON.parse(JSON.stringify(txs[4])), { state: 'ended' })" :validators="validators" unbonding_time="2592000000" v-on:end-unbonding="log('end unbonding')"/>
+      <tm-li-stake-transaction :transaction="txs[5]" :validators="validators" v-on:end-redelegation="log('end redelegation')"/>
```

Comment:
why do we need the end-redelegation event?

Reply:
Technically we need a BeginRedelegate and CompleteRedelegate:

https://github.com/cosmos/cosmos-sdk/blob/10b916eb28a8aed07df80d8c305a40b7f073d739/x/stake/types/msg.go#L204-L209

https://github.com/cosmos/cosmos-sdk/blob/10b916eb28a8aed07df80d8c305a40b7f073d739/x/stake/types/msg.go#L265-L270

Reply:
I'll open an issue on voyager for this